src/frontend/src/layouts/NavTopBar/NavUserMenu.tsx

```diff
@@ -48,6 +50,15 @@ const NavUserMenu: React.FC = () => {
     </MenuItem>
   );
 
+  const adminTools = (
+    <MenuItem component={RouterLink} to={routes.ADMIN_TOOLS} onClick={handleClose} sx={{ py: 0 }}>
+      <ListItemIcon>
+        <HomeRepairServiceIcon fontSize="small" />
+      </ListItemIcon>
+      <ListItemText>Admin Tools</ListItemText>
+    </MenuItem>
+  );
```

Comment:
rename this to AdminTools = () => (MenuItem...);
and then you can call it as <AdminTools/> down where you have adminTools
does the same thing, just looks a lil better in code